pkg/anago/release.go

```diff
@@ -91,6 +97,31 @@ func (d *defaultReleaseImpl) PrepareWorkspaceRelease(
 	return release.PrepareWorkspaceRelease(directory, buildVersion, bucket)
 }
 
+func (d *defaultReleaseImpl) CreateAnnouncement(
+	directory, buildVersion, releaseBranch string,
+) error {
+	// Get a semver from the buildVersion
+	semver, err := util.TagStringToSemver(buildVersion)
+	if err != nil {
+		return errors.Wrap(err, "parsing version tag into semver")
+	}
```

Comment:
This is already checked on options validation. Maybe we can store it there as private variable and add an assessor method to retrieve the semver-variant of the build version?

Reply:
Done: `options.SemVer()`

Reply:
:arrow_right: Now moved onto the `State{}` struct